public/stylesheets/detailed-results.css

```diff
@@ -104,27 +104,81 @@ body {
 
 #game-info-container {
   width: 100%;
+  min-height: 250px;
+  background-color: #333;
 }
 .detailed-results-text {
   flex-direction: column;
-  background-color: #333;
+  overflow-wrap: break-word;
+}
+.game-details-item {
+  font-size: 20px;
+  margin-bottom: 15px;
+}
+.details-text,
```

Comment:
This is a child element of `game-details-item` so it would be preferable if the naming followed the structure as well, per say: `game-details-text`?